bin/stacks/routing-caching-stack.ts

```diff
@@ -37,6 +39,11 @@ export class RoutingCachingStack extends cdk.NestedStack {
 
     const { chatbotSNSArn } = props
 
+    let chatBotTopic: cdk.aws_sns.ITopic | undefined
+    if (chatbotSNSArn) {
+      chatBotTopic = aws_sns.Topic.fromTopicArn(this, 'ChatbotTopic', chatbotSNSArn)
+    }
+
```

Comment:
nit: `const chatBotTopic = chatbotSNSArn ? aws_sns.Topic.fromTopicArn(this, 'ChatbotTopic', chatbotSNSArn) : undefined`

Reply:
> > Reduce retries for V2 because the way its currently set up (10 minutes * 3 retries = 30 minutes) is double the lambda maximum duration of 15 minutes.
> 
> Are we sure lambda retries don't count as separate invocations? (i.e. each retry has it's own 15-minute limit)?

sorry when I said this i didn't mean lambda retry, I meant subgraph retry: https://github.com/Uniswap/smart-order-router/blob/d0fdfb618656476bacfef802a21df51390614a40/src/providers/v2/subgraph-provider.ts#L69